.mergify.yml

```diff
@@ -0,0 +1,26 @@
+pull_request_rules:
+  - name: approve dependabot pull requests with build workflow
+    conditions:
+      - author~=dependabot\[bot\]
+      - files~=^(Dockerfile.*|\.github/workflows/build\.yml|package-templates/)
```

Comment:
Is `package-templates/` needed here since there's no dependabot config for that folder?

Reply:
Technically, probably not. But I thought it more clear to keep the files condition the same as the one that triggers the build workflow. There will be a little discrepancy either way, but both should work, either matching the build workflow, or matching the dependabot config.

Reply:
Okay.  That's good with me.